_includes/comment.html

```diff
@@ -13,7 +13,7 @@ <h3 class="comment__author" itemprop="author" itemscope itemtype="https://schema
     <p class="comment__date">
       {% if include.date %}
         {% if include.index %}<a href="#comment{{ include.index }}" itemprop="url">{% endif %}
-        <time datetime="{{ include.date | date_to_xmlschema }}" itemprop="datePublished">{{ include.date | date: "%B %d, %Y at %I:%M %p" }}</time>
+        <time datetime="{{ include.date | date_to_xmlschema }}" itemprop="datePublished">{{ include.date | date_to_xmlschema }}</time>
```

Comment:
`<time>` is already using `date_to_xmlschema` for the `datetime` attribute. My understanding the time between `<time></time>` is the human readable format and not used.

Reply:
You're [right](https://schema.org/docs/gs.html#advanced_dates) ! I reverted it.